bitcoind-regtest/test/Bitcoin/Core/Test/Wallet.hs

```diff
@@ -251,7 +251,9 @@ testDescriptorCommands v = do
                 Nothing
                 (Just "imported-descriptor")
             ]
-        when (v > v21_1) $ RPC.listDescriptors >>= shouldMatch 6 . length
+        when (v > v21_1 && v < v23_0) $ RPC.listDescriptors >>= shouldMatch 6 . length
+        -- Newly created descriptor wallets will contain an automatically generated tr() descriptor
+        when (v > v23_0) $ RPC.listDescriptors >>= shouldMatch 8 . length
```

Comment:
Should this be `>=`?  or perhaps the above should be `<=`?

Reply:
Yes good catch! I've updated it 👍🏼